### PR TITLE
Start containers for tests

### LINKED ISSUE #60: Orchestrator: Start containers with agents and shut them down when everything is done
* Create entity for agent: id, status, update time
* In `InitializeAgentsController.initialize`. Should build base image for execution, start a (configurable from properties) number of containers, create entries for agents in DB.
* When 'Everything is done' == there are no tests left in DB, all agents have IDLE status in DB and send corresponding heartbeats, stop all agents for this execution  <-- this can be extracted into a separate issue to make changes smaller

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Added logic in InitializeAgentsController
* Added entity for Agent
* Added LocalDateTimeSerializer
* Added agent table in liquibase
* Added AgentController in backend
* Fix: set ID for existing project too in CloneRepositoryController
* Fix: return executionId from createExecution handler
* Tests: added dependency mockito-kotlin

Closes #60 

### Todo:
* [x] tests
* [x] logic in backend
* [x] db changelogs for agents

### CODE REVIEW

File: db/changelog/v-1/07-changeset-agent-table.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="execution_id" type="bigint">
+                <constraints foreignKeyName="fk_agent_execution" references="execution(id)" nullable="false"/>
+            </column>
+        </createTable>
+
+        <addUniqueConstraint
+                tableName="agent"
+                columnNames="agent_id"/>
```
Comment (Cheshiriks) [inline db/changelog/v-1/07-changeset-agent-table.xml:25]: It's not needed. A primaryKey constraint includes a unique constraint

**Review by Cheshiriks — CHANGES_REQUESTED** [review]

File: save-backend/src/main/kotlin/org/cqfn/save/backend/repository/AgentStatusRepository.kt [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
-@Suppress("MISSING_KDOC_TOP_LEVEL")
+/**
+ * JPA repository for agent statuses.
+ */
 interface AgentStatusRepository : JpaRepository<AgentStatus, Long>
+
+/**
+ * JPA repository for agents.
+ */
+interface AgentRepository : JpaRepository<Agent, String>
```
[the remark this answers is not in the document]
Comment (petertrr) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/repository/AgentStatusRepository.kt:19]: Let's consider it in #85 along with other jpa enhancements :)

File: db/changelog/v-1/07-changeset-agent-table.xml (hunk 1 above)
Comment (petertrr) [reply db/changelog/v-1/07-changeset-agent-table.xml]: ok, removed

File: save-backend/src/main/kotlin/org/cqfn/save/backend/repository/AgentStatusRepository.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/repository/AgentStatusRepository.kt:19]: ок)

**Review by Cheshiriks — APPROVED** [review]

**Pull request merged**